### ISSUE TITLE
Error installing on Ubuntu 16.04

### ISSUE THREAD

**Issue by h3ct0r** [reedhedges/AriaCoda#1]
Hi!

Is this project compiling for Ubuntu 16.04? I have tried to compile with the normal: `make && sudo make install` and some errors arise:

```bash
--------------------------------------
Installing ARIA in /usr/local/Aria...
--------------------------------------
install --preserve-timestamps -m 775 -d /usr/local/Aria
find    include src tests utils params docs examples maps \
        java javaExamples python pythonExamples matlab rust \
        obj rust csharp csharpExamples \
        \( -name \*.o -or -name core -or -name CVS -or -name .\* -or -name \*~ -or -name tmp -or -name proprietary* -or -name \*.bak -or -name \*.class -or -name \*.lib -or -name \*.dll -or -name \*.exe \) -prune  \
        -or -type d   -exec install --preserve-timestamps -d -m 777 /usr/local/Aria/\{\} \; \
        -or -type l   -exec cp --no-dereference \{\} /usr/local/Aria/\{\} \; \
        -or -name \*.a -exec install --preserve-timestamps -D -m 666 \{\}  /usr/local/Aria/\{\} \; \
        -or -perm /u=x  -exec install --preserve-timestamps -D --strip -m 777 \{\}  /usr/local/Aria/\{\} \; \
        -or           -exec install --preserve-timestamps -D -m 666 \{\} /usr/local/Aria/\{\} \;
find: ‘csharp’: No such file or directory
find: ‘csharpExamples’: No such file or directory
Makefile:671: recipe for target 'install' failed
make: *** [install] Error 1
```

Then after commenting the csharp parts and redoing a `sudo make install` I have the following errors:

```bash
--------------------------------------
Installing ARIA in /usr/local/Aria...
--------------------------------------
install --preserve-timestamps -m 775 -d /usr/local/Aria
find    include src tests utils params docs examples maps \
        java javaExamples python pythonExamples matlab rust \
        obj rust \
        \( -name \*.o -or -name core -or -name CVS -or -name .\* -or -name \*~ -or -name tmp -or -name proprietary* -or -name \*.bak -or -name \*.class -or -name \*.lib -or -name \*.dll -or -name \*.exe \) -prune  \
        -or -type d   -exec install --preserve-timestamps -d -m 777 /usr/local/Aria/\{\} \; \
        -or -type l   -exec cp --no-dereference \{\} /usr/local/Aria/\{\} \; \
        -or -name \*.a -exec install --preserve-timestamps -D -m 666 \{\}  /usr/local/Aria/\{\} \; \
        -or -perm /u=x  -exec install --preserve-timestamps -D --strip -m 777 \{\}  /usr/local/Aria/\{\} \; \
        -or           -exec install --preserve-timestamps -D -m 666 \{\} /usr/local/Aria/\{\} \;
install --preserve-timestamps -D -m 664 LICENSE.txt INSTALL.txt README.txt Makefile Aria-Reference.html version.txt Changes.txt CommandLineOptions.txt icon.png /usr/local/Aria/
install: cannot stat 'INSTALL.txt': No such file or directory
install: cannot stat 'README.txt': No such file or directory
install: cannot stat 'Aria-Reference.html': No such file or directory
install: cannot stat 'version.txt': No such file or directory
install: cannot stat 'Changes.txt': No such file or directory
install: cannot stat 'CommandLineOptions.txt': No such file or directory
Makefile:671: recipe for target 'install' failed
make: *** [install] Error 1
```

Is this file 'CommandLineOptions.txt' important? Or what is the correct way to run/install this Aria version? 

This repo is very important since the original MobileRobots wiki is down and I could not find any information about if the site is comming back. I have called the Adept hotline and the know noting about the old Mobile Robots.

Thanks!

**Comment by h3ct0r** [comment]
Hi, @reedhedges thank you for your super quick response! I have made a pull request with small modifications of the makefile to allow the command `sudo make install` to work. :)
It is a shame that the old mobilerobots website is not working anymore (do you know where I can call/send email for that?), but I really thank you for having this repo public and updated!

**Comment by reedhedges** [comment]
This should be fixed by h3ct0r's PR #2  and my merge of that PR commit 5a3b17c3
